docs/xamarin-forms/xaml/xaml-previewer.md

```diff
@@ -22,20 +22,20 @@ There is more information in the [release notes](https://developer.xamarin.com/r
 
 ## Getting Started
 
-### Visual Studio for Mac on Mac
-
-The **Preview** button can be displayed on the editor by right-clicking a XAML file, and selecting **Open With > XAML Viewer**. The preview pane can then be shown or hidden by pressing the **Preview** button in the top-right corner of any XAML document window:
-
-[![ListView control preview in Visual Studio for Mac](xaml-previewer-images/xamlp-list-sml.png "Forms Previewer in Visual Studio for Mac")](xaml-previewer-images/xamlp-list.png#lightbox "Forms Previewer in Visual Studio for Mac")
-
-### Visual Studio on Windows
+# [Visual Studio](#tab/vswin)
 
 Use the **View > Other Windows > Xamarin.Forms Previewer** menu in Visual Studio
 to open the preview window. Use the **Window > New Vertical Tab Group**
 menu to position it side-by-side.
 
 [![ListView control preview in Visual Studio](xaml-previewer-images/xamlp-list-vs-sml.png "Forms Previewer in Visual Studio")](xaml-previewer-images/xamlp-list-vs.png#lightbox "Forms Previewer in Visual Studio")
 
+# [Visual Studio for Mac](#tab/vsmac)
+
+The **Preview** button can be displayed on the editor by right-clicking a XAML file, and selecting **Open With > XAML Viewer**. The preview pane can then be shown or hidden by pressing the **Preview** button in the top-right corner of any XAML document window:
+
+[![ListView control preview in Visual Studio for Mac](xaml-previewer-images/xamlp-list-sml.png "Forms Previewer in Visual Studio for Mac")](xaml-previewer-images/xamlp-list.png#lightbox "Forms Previewer in Visual Studio for Mac")
+
```

Comment:
@HankiDesign the tab markdown extension syntax requires a closing `-----` (five dashes) on a newline to indicate a return to 'normal' content. You'll need to insert that here (blank line before and after) otherwise the entire rest of the page will be in the "Mac" tab :)

Reply:
@conceptdev Thanks for the example, the fix is now committed.